sources/packages/api/src/route-controllers/confirmation-of-enrollment/confirmation-of-enrollment.controller.ts

```diff
@@ -133,54 +147,77 @@ export class ConfirmationOfEnrollmentController {
    * @returns application details for COE
    */
   @HasLocationAccess("locationId")
-  @Get(":locationId/confirmation-of-enrollment/application/:applicationId")
+  @Get(
+    ":locationId/confirmation-of-enrollment/disbursement/:disbursementScheduleId",
+  )
   async getApplicationForCOE(
     @Param("locationId") locationId: number,
-    @Param("applicationId") applicationId: number,
+    @Param("disbursementScheduleId") disbursementScheduleId: number,
```

Comment:
i think you missed to add `disbursementScheduleId` param in the comment

Reply:
comment updated.